static/app/components/events/eventReplay/replayClipSection.tsx

```diff
@@ -87,31 +115,13 @@ export function ReplayClipSection({event, group, replayId}: Props) {
     >
       <ErrorBoundary mini>
         <ReplayGroupContextProvider groupId={group?.id} eventId={event.id}>
-          <ReactLazyLoad debounce={50} height={448} offset={0} once>
-            <LazyLoad
-              analyticsContext="issue_details"
-              replaySlug={replayId}
-              orgSlug={organization.slug}
-              eventTimestampMs={eventTimestampMs}
-              fullReplayButtonProps={{
-                analyticsEventKey: 'issue_details.open_replay_details_clicked',
-                analyticsEventName: 'Issue Details: Open Replay Details Clicked',
-                analyticsParams: {
-                  ...getAnalyticsDataForEvent(event),
-                  ...getAnalyticsDataForGroup(group),
-                  organization,
-                },
-              }}
-              loadingFallback={
-                <StyledNegativeSpaceContainer data-test-id="replay-loading-placeholder">
-                  <LoadingIndicator />
-                </StyledNegativeSpaceContainer>
-              }
-              LazyComponent={ReplayClipPreview}
-              clipOffsets={REPLAY_CLIP_OFFSETS}
-              overlayContent={overlayContent}
-            />
-          </ReactLazyLoad>
+          {hasStreamlinedUI ? (
+            lazyReplay
+          ) : (
+            <ReactLazyLoad debounce={50} height={448} offset={0} once>
+              {lazyReplay}
+            </ReactLazyLoad>
+          )}
```

Comment:
I'm like pretty sure this component isn't necessary, but it doesn't work with `once` in the streamlined UI.